ethcore/src/engines/tendermint/mod.rs

```diff
@@ -260,8 +241,17 @@ impl Tendermint {
 		self.validators.contains(&*self.proposal_parent.read(), address)
 	}
 
-	fn is_above_threshold(&self, n: usize) -> bool {
-		n > self.validators.count(&*self.proposal_parent.read()) * 2/3
+	fn is_above_threshold(&self, n: usize) -> Result<(), EngineError> {
+		let threshold = self.validators.count(&*self.proposal_parent.read()) * 2/3;
```

Comment:
That won't really work for a single validator, right? If you ever reach to such state you will never be able to mint any new blocks.